app/scripts/views/settings/change_password.js

```diff
@@ -45,6 +45,7 @@ define(function (require, exports, module) {
           self.relier
         )
         .then(function () {
+          self.logEvent('settings.change_password.success');
```

Comment:
is it possible to add a unit test tracking this event? you might be be able to use an existing test and just add into it.